Modules/Scripted/WebServer/WebServerLib/SlicerRequestHandler.py

```diff
@@ -679,15 +693,45 @@ def setViewersLayout(layoutName):
         raise ValueError("Unknown layout name: " + layoutName)
 
     def gui(self, request):
-        """return a png of the application GUI.
-        :param contents: {full, viewers}
-        :param viewersLayout: {fourup, oneup3d, ...} slicer.vtkMRMLLayoutNode constants (SlicerLayout...View)
-        :return: png encoded screenshot after applying params
+        """Manage the application GUI.
+
+        - Path: /gui/version
```

Comment:
The `version` not being specific to the `gui`, what about `/application/version` ?

Reply:
Or since this call is already associated with the `/slicer` handler, having it under `/slicer/version` could work well ?